frontend/src/Context/PreviewPublisherProvider/usePreviewPublisher/usePreviewPublisher.tsx

```diff
@@ -228,10 +235,19 @@ const usePreviewPublisher = (): PreviewPublisherContextType => {
           }
         : undefined;
 
+    if (!isDeviceStoreReady) {
```

Comment:
Is there actually a scenario where this is true when called here if init hasn't been called?
or are we invoking this function twice? I'm a bit lost on the flow of this function